Scripts/Services/Expansions/High Seas/Spawners/BountyQuestSpawner.cs

```diff
@@ -664,19 +664,18 @@ public static void FillHold(BaseGalleon galleon)
 
                 if (Server.Engines.Points.PointsSystem.RisingTide.Enabled)
                 {
-                    if (0.25 > Utility.RandomDouble())
+                    hold.DropItem(new MaritimeCargo());
+                    hold.DropItem(new MaritimeCargo());
+
+                    if (galleon is OrcishGalleon)
                     {
                         hold.DropItem(new MaritimeCargo());
 
-                        if (0.1 > Utility.RandomDouble())
+                        if (Utility.RandomBool())
```

Comment:
Should there be a chance for a 4th cargo?  (3 or 4 from Pirate ships from [Heritage Bug](https://trueuo.com/threads/plunderbeacon-merchant-ship-cargo-not-dropping.2230/)

Reply:
There is 4. The first 2 are for all ship types. Then, orcish galleons (pirates) drop another, plus a chance for a fourth.

if (Server.Engines.Points.PointsSystem.RisingTide.Enabled)
                {
                    hold.DropItem(new MaritimeCargo());
                    hold.DropItem(new MaritimeCargo());

                    if (galleon is OrcishGalleon)
                    {
                        hold.DropItem(new MaritimeCargo());

                        if (Utility.RandomBool())
                        {
                            hold.DropItem(new MaritimeCargo());
                        }
                    }
                }